sdk/core/core-http/lib/policies/tracingPolicy.ts

```diff
@@ -31,7 +43,20 @@ export class TracingPolicy extends BaseRequestPolicy {
 
     // create a new span
     const tracer = getTracer();
-    const span = tracer.startSpan("core-http", request.spanOptions);
+    const spanOptions: SpanOptions = {
+      ...request.spanOptions,
+      kind: SpanKind.CLIENT
+    };
+    const span = tracer.startSpan("core-http", spanOptions);
+    span.setAttributes({
+      "http.method": request.method,
+      "http.url": request.url,
+      "http.requestId": request.requestId
```

Comment:
```suggestion
      "requestId": request.requestId
```

let's be consistent with other SDKs

Reply:
oops, this was a careless copypaste, thanks for catching it!